studio/pages/project/[ref]/settings/logs/[type].tsx

```diff
@@ -329,21 +332,19 @@ export const LogPage: NextPage = () => {
 
           {error && (
             <div className="flex w-full h-full justify-center items-center mx-auto">
-              <Card className="flex flex-col gap-y-2  w-1/3">
+              <Card className="flex flex-col gap-y-2  w-2/5 bg-scale-400">
                 <div className="flex flex-row gap-x-2 py-2">
                   <IconAlertCircle size={16} />
                   <Typography.Text type="secondary">
                     Sorry! An error occured when fetching data.
                   </Typography.Text>
                 </div>
-                <details className="cursor-pointer">
-                  <summary>
-                    <Typography.Text type="secondary">Error Message</Typography.Text>
-                  </summary>
-                  <Typography.Text className="block whitespace-pre-wrap" small code type="warning">
-                    {JSON.stringify(error, null, 2)}
-                  </Typography.Text>
-                </details>
+                <Input.TextArea
```

Comment:
we can replace this later with monaco style read only output, so it can be properly syntax highlighted